### PR TITLE
Fix language harvesting

### PR DESCRIPTION
**Pull request by fabianvf**

### CODE REVIEW

File: scrapi/base/helpers.py [hunk 1]
```diff
@@ -144,8 +144,16 @@ def pack(*args, **kwargs):
...
+
+    e.g. ['English', 'Russian'] -> ['eng', 'rus']
+    '''
+    return filter(lambda x: x, map(get_code, langs))
+
+
+def get_code(language):
     try:
-        return languages.get(name=language)
+        return languages.get(name=language).bibliographic
```
Comment (erinspace) [inline scrapi/base/helpers.py]: this key error is confusing - does .bibliographic access a list of some kind?

Comment (fabianvf) [inline scrapi/base/helpers.py]: no, but the ```.get(name=language)``` will throw a key error if it is not found.

Comment (erinspace) [inline scrapi/base/helpers.py]: coolcoolcool! Assumed languages was a dict, but I get it now!!

**Comment by fabianvf** [comment]
ping @erinspace

**Pull request merged**